src/main/java/com/brandon3055/draconicevolution/common/handler/ConfigHandler.java

```diff
@@ -428,6 +435,16 @@ public static void syncConfig() {
                 }
             }
 
+            itemDislocatorBlockBlacklistMap.clear();
+            for (String s : itemDislocatorBlockBlacklist) {
+                if (s.contains("|")) {
+                    itemDislocatorBlockBlacklistMap
+                            .put(s.substring(0, s.indexOf("|")), Integer.parseInt(s.substring(s.indexOf("|") + 1)));
```

Comment:
the parsing of the int will throw a `NumberFormatException`, best to catch it and log the error in the log